### PR TITLE
button is added to create new circuit direct from subcircuit dialog b…

### LINKED ISSUE #1751: Button to create "New Circuit" in "Insert Subcircuit" Dialogue
**Is your feature request related to a problem? Please describe.**
If there is only 1 circuit and we press ```Insert SubCircuit``` we get a dialogue box. Adding a button to ```Create a New Circuit``` can be helpful.

**Describe the solution you'd like**
A simply button or anchor tag to create a new circuit.

**Additional context**
Screenshot
image

**Are you working on this? (Yes/No)**
No.

### PR DESCRIPTION
**Pull request by GAUTAMSAHARAN**
Fixes #1751

### CODE REVIEW

**Comment by GAUTAMSAHARAN** [comment]
Screenshot from 2020-10-04 17-48-46

File: public/css/main.stylesheet.css [hunk 1]
```diff
@@ -841,7 +841,7 @@ input:checked + .slider:before {
   border-radius: 5px;
   width: 600px;
   height: 320px;
-  padding: 10px 17px;
+  padding: 10px 17px !important;
```
Comment (abstrekt) [inline public/css/main.stylesheet.css:844]: Why is this necessary?

**Review by abstrekt — CHANGES_REQUESTED** [review]
Avoid inline style,
is it not possible to do with a single div? instead of having a nonEmpty and an empty Div?
Also, have the same button style as the insertSubcircuit button
Use proper HTML breaks instead of having one long line of HTML
After creating a new circuit through the new circuit button, update the div with the appropriate options.

File: app/views/simulator/edit.html.erb [hunk 2]
```diff
@@ -489,7 +489,7 @@
   id="insertSubcircuitDialog"
   class="subcircuitdialog"
   title="Insert SubCircuit"
-></div>
+><div id="subCircuitNonEmptyDiaglog" style="height: fit-content;"></div><div id="subCircuitEmptyDialog"><span><p>Looks like there are no other circuits which doesn't have this circuit as a dependency. Create a new one!</p></span><a style="color:white;" class="btn btn-xs custom-btn--primary dropdown-item text-center logixButton pl-1" id="newCircuit">New Circuit +</a></div></div>
```
Comment (satu0king) [inline app/views/simulator/edit.html.erb:492]: Creating a new dialog for nonEmpty and Empty is over kill. Please use old dialog itself.

**Comment by tachyons** [comment]
Closing due to inactivity

**Pull request closed without merging**